### PR TITLE
Store the budget

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- load the expert flow
- scroll down the budget
- enter a budget and click the button
- the rest of the flow should appear

**Comment by huwshimi** [comment]
[Store the expert budget](https://trello.com/c/QAqHYL0i/582-store-the-expert-budget)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8656/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8657/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8658/

File: jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js [hunk 1]
```diff
@@ -676,7 +676,9 @@ class DeploymentFlow extends React.Component {
         instance="deployment-expert-budget"
         showCheck={true}
         title="Set your maximum monthly budget (optional)">
-        <DeploymentExpertBudget />
+        <DeploymentExpertBudget
+          budget={this.state.budget}
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js]: budgetValue ?

**Comment by hatched** [comment]
Thanks @huwshimi :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**